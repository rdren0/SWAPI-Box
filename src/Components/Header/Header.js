import React from 'react';

const Header = () => {

  return(
    <div>
      <h1> SWAPI BOX </h1>
      <br/>
      <button className="favorite-button"> Favorites </button>
    </div>
    )

}

export default Header;