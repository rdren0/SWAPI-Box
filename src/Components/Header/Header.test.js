import React, { Component } from 'react';
import Header from './Header.js';
import { shallow } from 'enzyme';



describe(Header, () => {
  it('should match snapshot with all correct data passed through', () => {
    const wrapper = shallow(
      <Header/>)
    expect(wrapper).toMatchSnapshot();
  })
})