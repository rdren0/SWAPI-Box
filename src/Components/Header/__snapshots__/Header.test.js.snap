// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match snapshot with all correct data passed through 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Header />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1>
           SWAPI BOX 
        </h1>,
        <br />,
        <button
          className="favorite-button"
        >
           Favorites 
        </button>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": " SWAPI BOX ",
        },
        "ref": null,
        "rendered": " SWAPI BOX ",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": "br",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": " Favorites ",
          "className": "favorite-button",
        },
        "ref": null,
        "rendered": " Favorites ",
        "type": "button",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
             SWAPI BOX 
          </h1>,
          <br />,
          <button
            className="favorite-button"
          >
             Favorites 
          </button>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": " SWAPI BOX ",
          },
          "ref": null,
          "rendered": " SWAPI BOX ",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": "br",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": " Favorites ",
            "className": "favorite-button",
          },
          "ref": null,
          "rendered": " Favorites ",
          "type": "button",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": true,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
